src/components/Toggle/index.tsx

```diff
@@ -44,7 +44,7 @@ const Toggle = <T extends {}>(props: Props<T>) => {
                 category='neutral'
                 onClick={() => onChange(item)}
                 data-testid={`toggle-element-${getLabel(item)}`}
-                variablesClassName={classNames(styles['toggle-span'])}
+                variablesClassName={classNames(styles['toggle-span'], variablesClassName)}
```

Comment:
hey @branquinhoaa could you please add an example showing a custom-styled toggle component?